packages/examples/cvat/exchange-oracle/src/handlers/job_creation.py

```diff
@@ -2091,7 +2253,19 @@ def _mangle_filenames(self):
             roi_info.bbox_id: str(uuid.uuid4()) + self.roi_file_ext for roi_info in self._roi_infos
         }
 
-    def _prepare_job_params(self):
+        self._input_filename_to_nameless = {
+            roi_info.original_image_id: self._roi_filenames[roi_info.bbox_id]
+            for roi_info in self._roi_infos
+        }
+
+    @property
+    def _task_segment_size(self):
+        # Unlike other task types, here we use a grid of RoIs,
+        # so the absolute job size numbers from manifest are multiplied by the job size multiplier.
```

Comment:
```suggestion
        ```suggestion
        # Here we use a job size multiplier, because each image 
        # is supposed to be simple and the assignment is expected 
        # to take little time with the default job size.
```
```